src/main/sensors/battery.c

```diff
@@ -50,6 +50,8 @@
 
 #define ADCVREF 3300   // in mV
 
+#define MAX_ESC_BATTERY_AGE 10
```

Comment:
I don't like the "AGE" as it relates to time relation with something, while in this case it is simply amount of max retries